### DISCUSSION TITLE
How to change the default port number of MONAI Label server in OHIF

### DISCUSSION THREAD

**Discussion by tcolliri** [Project-MONAI/MONAILabel#1059] [Q&A] [answered]
Hi everyone. 
I have been reading a lot of information available online about MONAI Label, and I was already able to successfully apply the segmentation2D app to DICOM data from mammograms, using MONAI Label, Orthanc and OHIF, all of them integrated and functioning well. 
However, right now I am really struggling to find a way of changing the default port number of MONAI Label server in OHIF. I mean, when I click on the 'MONAI Label' option, on the right side of the screen in OHIF, there is the default value for the 'Server' field, which is always: 'http://localhost:8000/'. So how can I change the default port number in this address? 
I would like to set a different port as the default in this field, such as 8001 or 8002, for example. Then I would be able to use this different port number in combination with the option '--port 8001', for example, when starting the MONAI Label server.  
I really appreciate your help.
Thank you.

**Comment by diazandr3s** [comment, marked as answer]
Hi @tcolliri,

Thanks for opening this discussion.
I've checked this use case and you may need to refress the page after changing the port manually. Please watch this video: 

https://user-images.githubusercontent.com/11991079/194979203-e3fe6132-c2c1-4072-92ab-35cba98dd64e.mp4

Hope this helps

**Reply by tcolliri** [reply]
Hi Andres,
Thank you very much for your reply.
By the way, let me mention that I was only capable of successfully implementing MONAI Label because I have spent several days here reading online discussions with you responding to lots of users around the internet. So, a big THANK YOU for your time and patience with everyone. And congratulations for your amazing work.
